internal/cmdmigrate/migratecmd_test.go

```diff
@@ -177,6 +178,118 @@ func TestKptMigrate_updateKptfile(t *testing.T) {
 	}
 }
 
+func TestKptMigrate_migrateKptfileToRG(t *testing.T) {
+	testCases := map[string]struct {
+		kptfile       string
+		rgFilename    string
+		resourcegroup string
+		dryRun        bool
+		isError       bool
+	}{
+		"Missing Kptfile is an error": {
+			kptfile:    "",
+			rgFilename: "resourcegroup.yaml",
+			dryRun:     false,
+			isError:    true,
+		},
+		"Kptfile with existing inventory will create ResourceGroup": {
```

Comment:
So in this situation, do we only move the inventory information from the Kptfile into the rg file without attempting any update of the live state? At first it seems like we maybe wouldn't care about the live state, but it might be that a previous run failed midway through, so we should verify the live state and complete any in-process migrations.

Reply:
Synced up on this offline. For existing Kptfiles that contain an inventory, the cluster's inventory should be tracking all resources as a ResourceGroup CR on the cluster. If there is drift between the cluster state and the local config, the differences will be reconciled when the user next runs `kpt live apply`.
For a ConfigMap migration, this new migrate CUJ still follows the existing migration flow as from CM to Kptfile. The only difference is that we write the inventory to `resourcegroup.yaml` instead of the local Kptfile.